lazy_tensor_core/test/cpp/test_aten_ltc_ts_tensor.cpp

```diff
@@ -6163,6 +6163,22 @@ TEST_F(AtenLtcTsTensorTest, TestLogSoftmaxCast) {
   });
 }
 
+TEST_F(AtenLtcTsTensorTest, TestLogSoftmaxWrapper) {
```

Comment:
We already have a test, `TestLogSoftmax`. Why do we care about testing the `_log_softmax` directly?

Reply:
Just mimic what the TestSoftmaxWrapper does.